metadata.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://www.snowplowanalytics.com"
+documentation: "https://docs.snowplowanalytics.com/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/"
```

Comment:
I'd suggest we launch the docs page so we can update this URL to the specific page before merging the tag.